tools/font-edit/README.md

```diff
@@ -14,4 +18,66 @@ make
 ./twin-fedit < nchars
 ```
 
-(press 'q' to next character)
+### Background
```

Comment:
Don't dump the source code.

Instead, mention that the glyphs originated from [Hershey vector fonts](https://en.wikipedia.org/wiki/Hershey_fonts) and were fitted with splines for curved portions, along with the concepts used in this font editor.

Reply:
I found a short video [The Hershey Fonts — Frank Grießhammer](https://youtu.be/xQNHAWrR_eg?si=onOu-ipeI53H4xzl) to introduce the history of the Hershey fonts by the Adobe font designer Frank Grießhammer.

An example of drawing the character 'H' is mentioned in [Hershey Vector Font Based on the hershey character set by Paul Bourke](https://paulbourke.net/dataformats/hershey/), along with the MSWindows .fnt font contributed by Gary Gofstein.

I will organize this information and then update the document accordingly.